pkg/option/config.go

```diff
@@ -260,6 +260,10 @@ const (
 	// ToFQDNsEmitPollerEvents controls if poller lookups are sent as monitor events
 	ToFQDNsEnablePollerEvents = "tofqdns-enable-poller-events"
 
+	// ToFQDNsMaxIPsPerHost defines the maximum number of IPs to maintain
+	// for each FQDN name in an endpoint's FQDN cache
+	ToFQDNsMaxIPsPerHost = "tofqdns-max-ip-per-hostname"
```

Comment:
`tofqdns-endpoint-max-ip-per-hostname` might be a little more clear but it is longer.